### PR TITLE
Moving block pool cypress testcases to ODF console

### PR DESCRIPTION
**Pull request by GowthamShanmugam**
Signed-off-by: Gowtham Shanmugasundaram <gshanmug@redhat.com>

### CODE REVIEW

**Comment by bipuladh** [comment]
Let's not remove the tests until we completely move the code from here to ODF. Please update the data-test only.

**Comment by GowthamShanmugam** [comment]
> Let's not remove the tests until we completely move the code from here to ODF. Please update the data-test only.

ack

File: frontend/packages/ceph-storage-plugin/integration-tests-cypress/tests/block-pool-create.spec.ts [hunk 1]
```diff
@@ -1,7 +1,8 @@
 import { checkErrors } from '../../../integration-tests-cypress/support';
 import { createBlockPool, deleteBlockPoolFromCli } from '../views/block-pool';
 
-describe('Test block pool creation under OCS UI', () => {
+/** @depricated testcases are moved to odf-console */
+xdescribe('Test block pool creation under OCS UI', () => {
```
Comment (bipuladh) [inline frontend/packages/ceph-storage-plugin/integration-tests-cypress/tests/block-pool-create.spec.ts:5]:
```suggestion
describe('Test block pool creation under OCS UI', () => {
```

File: frontend/packages/ceph-storage-plugin/integration-tests-cypress/tests/block-pool-delete.spec.ts [hunk 2]
```diff
@@ -13,7 +13,8 @@ import { createStorageClass } from '../views/storage-class';
 
 const pvcName: string = 'testing-pvc';
 
-describe('Test block pool deletion under OCS UI', () => {
+/** @depricated testcases are moved to odf-console */
+xdescribe('Test block pool deletion under OCS UI', () => {
```
Comment (bipuladh) [inline frontend/packages/ceph-storage-plugin/integration-tests-cypress/tests/block-pool-delete.spec.ts:17]:
```suggestion
describe('Test block pool deletion under OCS UI', () => {
```

File: frontend/packages/ceph-storage-plugin/integration-tests-cypress/tests/block-pool-update.spec.ts [hunk 3]
```diff
@@ -9,7 +9,8 @@ import {
   poolMessage,
 } from '../views/block-pool';
 
-describe('Test block pool update under OCS UI', () => {
+/** @depricated testcases are moved to odf-console */
+xdescribe('Test block pool update under OCS UI', () => {
```
Comment (bipuladh) [inline frontend/packages/ceph-storage-plugin/integration-tests-cypress/tests/block-pool-update.spec.ts:13]:
```suggestion
describe('Test block pool update under OCS UI', () => {
```

File: frontend/packages/ceph-storage-plugin/integration-tests-cypress/tests/multiple-pool.spec.ts [hunk 4]
```diff
@@ -36,7 +36,7 @@ const createBlockPool = (poolCreationAction: string) => {
   verifyBlockPoolJSON();
 };
 
-// Todo(bipuladh): Enable after downstream builds are available with v1 CSIDrivers
+/** @depricated testcases are moved to odf-console */
 xdescribe('Test Ceph pool creation', () => {
```
Comment (bipuladh) [inline frontend/packages/ceph-storage-plugin/integration-tests-cypress/tests/multiple-pool.spec.ts:40]:
```suggestion
describe('Test Ceph pool creation', () => {
```

**Review by bipuladh — COMMENTED** [review]
Let's not disable them until we move to 4.11 for ODF.

**Comment by bipuladh** [comment]
/approve
/lgtm
/px-approved
/qe-approved
/docs-approved
Test changes only

File: frontend/packages/console-shared/src/components/dashboard/details-card/DetailItem.tsx [hunk 5]
```diff
@@ -20,8 +20,15 @@ export const DetailItem: React.FC<DetailItemProps> = React.memo(
...
     return (
       <>
-        <dt className="co-details-card__item-title">{title}</dt>
-        <dd className={classNames('co-details-card__item-value', valueClassName)}>{status}</dd>
+        <dt className="co-details-card__item-title" data-test="detail-item-title">
+          {title}
+        </dt>
+        <dd
+          className={classNames('co-details-card__item-value', valueClassName)}
+          data-test="detail-item-value"
```
Comment (bipuladh) [inline frontend/packages/console-shared/src/components/dashboard/details-card/DetailItem.tsx:28]: @rawagner PTAL

**Pull request merged**